Documentation/tpv34.rst

```diff
@@ -0,0 +1,168 @@
+
+   Diagram of TPV34. 
+
+The geometry is generated with `Gmsh <https://gmsh.info/>`_. All the files 
+that are needed for the simulation are provided at 
+https://github.com/SeisSol/Examples/tree/master/tpv34.
+
+.. figure:: LatexFigures/tpv34_gmshhalf.png
+   :alt: Diagram of TPV34 vertical fault in Gmsh.
+   :width: 15.00000cm
+
+   Diagram of TPV34 vertical fault in Gmsh.
+
+Material
+~~~~~~~~
+
+To obtain the velocity structure for TPV34, we need to install and run 
+the SCEC Community Velocity Model `CVM-H <https://strike.scec.org/scecpedia/CVM-H>`_. 
+For TPV34, we are using CVM-H version 15.1.0 and we use 
+`ASAGI <https://github.com/TUM-I5/ASAGI>`_ to map related material properties 
+for a SeisSol simulation. Detailed explanations are provided at 
+https://github.com/SeisSol/Examples/blob/master/tpv34/generate_ASAGI_file.sh
+
+.. code-block:: YAML
+
+  [rho, mu, lambda]: !IdentityMap
+    components:
+      # apply a finer CVM-H data inside the refinement zone
+      - !AxisAlignedCuboidalDomainFilter
+        limits:
+          x: [-25000.0, 25000.0]
+          y: [-25000.0, 25000.0]
+          z: [-25000.0, 0.0]
+        components: !ASAGI
+          file: tpv34_rhomulambda-inner.nc
+          parameters: [rho, mu, lambda]
+          var: data
+      # apply a coarser CVM-H data outside the refinement zone
+      - !ASAGI
+          file: tpv34_rhomulambda-outer.nc
+          parameters: [rho, mu, lambda]
+          var: data
+
+
+Parameters
+~~~~~~~~~~
+
+TPV34 uses a linear slip weakening law on the fault. The parameters are
+listed in Table below.
+
++-------------+--------------------------------+------------+--------+
+| Parameter   | inside the nucleation zone     | Value      | Unit   |
++=============+================================+============+========+
+| mu\_s       | static friction coefficient    | 0.58       |        |
++-------------+--------------------------------+------------+--------+
+| mu\_d       | dynamic friction coefficient   | 0.45       |        |
++-------------+--------------------------------+------------+--------+
+| d\_c        | critical distance              | 0.18       | m      |
++-------------+--------------------------------+------------+--------+
+
+The cohesion is 1.02 MPa at the earth’s surface. It is 0 MPa at depths 
+greater than 2400 m, and is linearly tapered in the uppermost 2400 m.
+
+.. code-block:: YAML
+
+  [mu_d, mu_s, d_c]: !ConstantMap
+    map:
+      mu_d: 0.45
+      mu_s: 0.58
+      d_c: 0.18
+  [cohesion]: !FunctionMap
+    map:
+      cohesion: |
+        return -425.0*max(z+2400.0,0.0);
+
+Initial stress
+~~~~~~~~~~~~~~
+
+The initial shear stress on the fault is pure right-lateral. 
+The initial shear stress is :math:`\tau_0` = (30 MPa)(mux) 
+The initial normal stress on the fault is :math:`\sigma_0` = (60 MPa)(mux).
+With mux = :math:`\mu / \mu_0`, where 
+:math:`\mu` is shear modulus and
+:math:`\mu_0` = 32.03812032 GPa.
+
+.. code-block:: YAML
```

Comment:
In this PR. let's leave it like that, but why not using a 2d Netcdf file with the mu variation, and then generate Ts, Td and Pn based on that (see e.g. Norcia files for a model)

Reply:
I think this is fine since this setup made following the SCEC description.